tests/php/Control/RequestHandlingTest/HTTPMethodTestController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace SilverStripe\Control\Tests\RequestHandlingTest;
+
+use SilverStripe\Control\Controller;
+use SilverStripe\Control\HTTPRequest;
+
+class HTTPMethodTestController extends Controller
```

Comment:
It might be worth adding `implements TestOnly`, even though it's not a DataObject it still makes it clear

Reply:
Nice catch! Fixed.